### ISSUE TITLE
ts-morph panics when a broken symlink is anywhere in the repo

### ISSUE THREAD

**Issue by phiresky** [dsherret/ts-morph#827]
ts-morph 7.1.0 

I have a symlink in my code repo that only works after some process unrelated to typescript runs. The symlink is not anywhere near the code, but it is in a directory below the tsconfig.json

**To Reproduce**

1. Create a ts project configured with exclude: [..], i.e. one that uses a blacklist not a whitelist of files.
2. Create any broken symlink anywhere in the repository, e.g. `ln -s foo bar`
3. Load the project using ts-morph

```
(node:740546) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, stat '.../backup.log'
    at Object.realpathSync (fs.js:1674:9)
    at RealFileSystemHost.realpathSync (node_modules/@ts-morph/common/dist/ts-morph-common.js:1062:24)
    at TransactionalFileSystem.realpathSync (node_modules/@ts-morph/common/dist/ts-morph-common.js:1618:65)
    at node_modules/@ts-morph/common/dist/ts-morph-common.js:2318:38
    at visitDirectory (/node_modules/typescript/lib/typescript.js:17207:45)
```

Since this does not happen with tsc and ts-node, I'm guessing the realpathSync function should be catching this error?

**Comment by dsherret** [comment]
@phiresky thanks! You're right. The ts compiler uses a try/catch, but ts-morph does not:

https://github.com/microsoft/TypeScript/blob/8136047a3d9c32893508caf6ee51321545b92804/src/compiler/sys.ts#L1690

I'll fix this.

**Comment by dsherret** [comment]
@phiresky should be fixed in 7.1.2. Let me know if not.
